cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -73,16 +101,16 @@ def import_handler(request, course_key_string):
             # Do everything in a try-except block to make sure everything is properly cleaned up.
             try:
                 data_root = path(settings.GITHUB_REPO_ROOT)
-                course_subdir = "{0}-{1}-{2}".format(course_key.org, course_key.course, course_key.run)
-                course_dir = data_root / course_subdir
+                subdir = base64.urlsafe_b64encode(repr(courselike_key))
```

Comment:
Right! Sorry - I thought this was a directory in a constructed course tarball. I now see it's a directory for an uploaded course tarball to be saved.